libpam_misc/Makefile.am

```diff
@@ -20,3 +20,8 @@ libpam_misc_la_LIBADD = $(top_builddir)/libpam/libpam.la
 lib_LTLIBRARIES = libpam_misc.la
 
 libpam_misc_la_SOURCES = help_env.c misc_conv.c
+
+# Pkg-config script.
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = pam_misc.pc
+DISTCLEANFILES = $(pkgconfig_DATA)
```

Comment:
Since files created by `AC_CONFIG_FILES` are removed by `make distclean` automatically, no `DISTCLEANFILES` is needed.